tools/kvrocks2redis/parser.cc

```diff
@@ -323,9 +331,10 @@ rocksdb::Status WriteBatchExtractor::DeleteCF(uint32_t column_family_id, const S
         break;
       }
       case kRedisSortedint: {
-        if (!to_redis_) {
-          command_args = {"SIREM", user_key, std::to_string(DecodeFixed64(sub_key.data()))};
+        std::string sub_key_str=std::to_string(DecodeFixed64(sub_key.data()));
+        cmd_str = to_redis_ ? "SREM" : "SIREM";
```

Comment:
there is a compiling error, without definition. Please fix it @fishery86

Reply:
And change `SREM` to `ZREM`